tools/build/android/webrtc-native/build.gradle

```diff
@@ -17,7 +17,7 @@ android {
         }
         externalNativeBuild {
             cmake {
-                cppFlags "-std=c++14 -DANDROID_STL=c++_shared -DMRS_USE_STR_WRAPPER -DMR_SHARING_ANDROID -DWEBRTC_POSIX -DWEBRTC_ANDROID"
+                cppFlags "-std=c++14 -DANDROID_STL=c++_shared -DMRS_USE_STR_WRAPPER -DMR_SHARING_ANDROID -DUNITY_ANDROID -DWEBRTC_POSIX -DWEBRTC_ANDROID -DSUPPORT_OPENGL_ES=1" 
```

Comment:
Why do you need to add these defines to this lib?

Reply:
These are needed for the unity graphics interfaces, it doesn't compile if they are removed.

Reply:
But this file is used for mrwebrtc which doesn't include the Unity graphics interfaces, right?